src/back/project/src/model/cc_type_param_serialize.cpp

```diff
@@ -0,0 +1,31 @@
+#include "cc_type_param_serialize.hpp"
+
+#include <userver/formats/json/value_builder.hpp>
+
+namespace svetit::project::model {
+
+formats::json::Value Serialize(
+	const CcTypeParam& ccTypeParam,
```

Comment:
Давай везде в таких типовых методах использовать одинаковое обобщённое имя (например item или другое), чтобы бы проще копипастить.
```suggestion
	const CcTypeParam& item,
```

Reply:
Готово.